contracts/modules/cover/CoverNFT.sol

```diff
@@ -38,7 +38,23 @@ contract CoverNFT is ERC721 {
   }
 
   function operatorTransferFrom(address from, address to, uint256 tokenId) external onlyOperator {
-    super.transferFrom(from, to, tokenId);
+        require(from == _ownerOf[tokenId], "WRONG_FROM");
+
+        require(to != address(0), "INVALID_RECIPIENT");
+
+        // Underflow of the sender's balance is impossible because we check for
+        // ownership above and the recipient's balance can't realistically overflow.
+        unchecked {
+            _balanceOf[from]--;
+
+            _balanceOf[to]++;
+        }
+
+        _ownerOf[tokenId] = to;
+
+        delete getApproved[tokenId];
+
+        emit Transfer(from, to, tokenId);
```

Comment:
nitpick: can we remove some of the extra new lines? ex:

```solidity
        require(from == _ownerOf[tokenId], "WRONG_FROM");
        require(to != address(0), "INVALID_RECIPIENT");

        // Underflow of the sender's balance is impossible because we check for
        // ownership above and the recipient's balance can't realistically overflow.
        unchecked {
            _balanceOf[from]--;
            _balanceOf[to]++;
        }

        _ownerOf[tokenId] = to;
        delete getApproved[tokenId];

        emit Transfer(from, to, tokenId);
```